app/client/packages/mcp/.env.example

```
@@ -0,0 +1,63 @@
+# The MCP server, its data layer, and restricted JS objects are all ON by default in shipped deployments
+# (docker.env.sh sets each =true, and entrypoint.sh backfills them on upgrade). That default is intentional: the
+# authenticated MCP surface is a first-class CE feature, and the data-layer security review it was gated behind is
+# complete (it is this body of work). Each gate remains an Admin off-switch — set a var below to 0/false to disable
+# that layer for an instance. This file is the template for running the package standalone in local development.
+APPSMITH_MCP_ENABLED=1
+APPSMITH_MCP_PORT=8092
+APPSMITH_API_BASE_URL=http://127.0.0.1:8080
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APPSMITH_API_BASE_URL key should go before the APPSMITH_MCP_ENABLED key

(UnorderedKey)